Frameworks/UIKit/WYPopoverController.m

```diff
@@ -1459,10 +1457,15 @@ - (CGRect)innerRect:(CGRect)rect arrowDirection:(WYPopoverArrowDirection)aArrowD
         result.size.height -= borderWidth;
     }
     
+    // The below code is disabled on WINOBJC due to an issue where |viewContentInsets| has a garbage value (in DEBUG builds).
+    // Given that most of the theming/drawing code in WYPopoverController is effectively a no-op in WINOBJC,
+    // just skipping over the below is an adequate (or at least quick) fix.
```

Comment:
Can you please file a GitHub issue and add a TODO that references it here?  @Dhowett-MSFT can probably help track down the issue with kvo.